tasks.py

```diff
@@ -113,16 +102,16 @@ def build_image(
         force_rm (bool): Always remove intermediate containers
         hide: (bool): Suppress output from docker build
         with_ansible (bool): Build a container with Ansible installed
-        with_ansible_base (bool): Build a container with ansible-base installed
```

Comment:
Maybe out of scope for this PR, but at some point, the task args should be moved out of the docstring and into the `@task` decorator, like:

```python
@task(
    help={
        "cache": "Whether to use the Docker cache when building the image",
        "force_rm": "Always remove intermediate containers",
        ...
    }
)
def build_image(
```